### PR TITLE
Multiple variants in same codon

### PR DESCRIPTION
**Pull request by jonca79**
Adds multiple variant in the vcf. Uses information from the variant with the lowest AF.

### CODE REVIEW

File: src/Snakemake/rules/VCF_fix/Multibp_SNV.smk [hunk 1]
```diff
@@ -4,8 +4,19 @@ rule Find_multibp_SNV:
...
+
+rule sort_multiplebp_vcf:
+    input:
+        vcf="recall/{sample}.ensemble.vep.exon.soft_filter.multibp.vcf.temp",
+    output:
+        vcf="recall/{sample}.ensemble.vep.exon.soft_filter.multibp.vcf",
+    singularity:
+        config["singularity"].get("bcftools", config["singularity"].get("default", ""))
+    shell:
+        "bcftools sort -o {output.vcf} -O v {input.vcf}"
```
Comment (Smeds) [inline src/Snakemake/rules/VCF_fix/Multibp_SNV.smk]: Maybe use a [wrapper](https://snakemake-wrappers.readthedocs.io/en/0.68.0/wrappers/bcftools/sort.html)? With it you can easily change output format

**Review by Smeds — CHANGES_REQUESTED** [review]
Just a suggestion to change to a wrapper!

File: src/Snakemake/rules/VCF_fix/Multibp_SNV.smk (hunk 1 above)
Comment (jonca79) [reply src/Snakemake/rules/VCF_fix/Multibp_SNV.smk]: Added a wrapper

**Comment by Smeds** [comment]
I removed the commented lines so that snakefmt would pass! Now you have to review it @jonca79 =)

**Review by Smeds — APPROVED** [review]

**Pull request merged**